README.md

```diff
@@ -0,0 +1,67 @@
+# Conduit Connector Vitess
+
+## General
+
+The [Vitess](https://vitess.io/) connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and a destination Vitess connector.
+
+### Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- [MySQL](https://www.mysql.com/) versions 5.7 to 8.0 with binlog enabled
+- [Vitess](https://vitess.io/docs/14.0/get-started/) v14.0
+- [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/)
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint)
+
+### How to build it
+
+Run `make build`.
+
+### Testing
+
+Run `make test` to run all the unit and integration tests, which require Docker and Compose V2 to be installed and running. The command will handle starting and stopping docker containers for you.
+
+## Source
+
+The Vitess Source Connector connects to a VTGate with the provided `address`, `keyspace` and `tabletType` and starts creating records for each change detected in a table.
+
+Upon starting, the source takes a snapshot of a given table in the database, then switches into CDC mode. In CDC mode, the plugin reads events from a VStream. In order for this to work correctly, binlog must be enabled on MySQL instances you use with Vitess.
+
+### Snapshot Capture
+
+When the connector first starts, snapshot mode is enabled. The connector reads all rows of a table in batches via SELECT with [keyset pagination](https://use-the-index-luke.com/no-offset) by `orderingColumn`, limiting the rows by `batchSize` and ordering them by the `orderingColumn`. Once all rows in that initial snapshot are read the connector switches into CDC mode.
+
+This behavior is enabled by default, but can be turned off by adding `"snapshot": false` to the Source configuration.
+
+### Change Data Capture
```

Comment:
Are changed which happen **during a snapshot** captured?

Reply:
No, they're not. I've added a note about this.